templates/AddNewCoreLibrary.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+    .SYNOPSIS
+    Adds the class library and NuGet projects necessary to support a new core library
+    to the solution.
+
+    .PARAMETER CoreLibraryName
```

Comment:
I guess it is changed in all places it is needed because I used it to create Windows.ShellScalingApi after renaming all references of it.

By the way, I disabled on this script the part that looks for a backing DLL and generates the .txt files but I am afraid it might be used somewhere else because of the MSBuild things and/or StyleCop things. I guess I have seen references to that when I used this templating engine on another projects ([WinApi@github](https://github.com/arlm/WinApi) and [Extensions@github](https://github.com/arlm/Extensions)). Hope you don't mind I kind of "stole the idea" and ported it to these other projects. I tried to reach you on gitter before doing that but couldn't reach you.